esi_leap/common/idp/dummyIDP.py

```diff
@@ -0,0 +1,36 @@
+from oslo_utils import uuidutils
+from keystoneclient.v3.projects import Project, ProjectManager
+
+from esi_leap.common import exception
+from esi_leap.common.idp import baseIDP
+
+dummy_project = Project(manager=ProjectManager, info={
+    'id' : 1,
```

Comment:
Ids should be valid UUIDs; some functions validate their input and will throw warnings or errors if passed a non-UUID value.

Reply:
Ideally, we should be able to provision the dummy data externally. E.g., in tests we should be able to create a dictionary of projects and users and pass that in to the IDP so that we get known responses when making requests to the IDP.

Reply:
@larsks Do you want me to add functions in `dummyIDP.py` or `baseIDP.py` to add projects or users?